dmscripts/helpers/date_helpers.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime, timedelta
+from dmutils.formats import DATE_FORMAT
+
+
+def get_date_closed(date_closed):
```

Comment:
This is quite a generic name, I'd consider putting something in the docstring about the default and why it's yesterday or maybe even something like:
```
def get_date_closed(date_closed, default=None):
   return datetime.strptime(date_closed, DATE_FORMAT).date() if date_closed else default
```
Then when you call it pass in the default.

Reply:
I'll add an explanation in the docstring - it feels a bit weird adding a `default=None` param when `date_closed` itself might be `None` anyway.